### ISSUE TITLE
Graccu's Mince Meat Fruitcake

### ISSUE THREAD

**Issue by Lafadin** [dalaranwow/dalaran-wow#5186]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

I (Lafadin) was in Org bank right after completing the Metzen quest needed for the Merrymaker achievement (and ultimately that beautiful violet mount) earlier today (12/16/2017). I opened the bag and got no loot window nor anything from the bag. I didn't think anything of it, thinking I had missed something unimportant.  Upon trying to accomplish the "Tis' the Season" achievement about an hour or two later (around 3:00 US Mountain Time), I realized that Graccu's Mince Meat Fruitcake was supposed to drop from the bag; but I never received it.

**Expected behaviour**: Tell us what should happen instead

I should have received Graccu's Mince Meat Fruitcake in the bag received from the Metzen rescuing quest for the Merrymaker achievement. I was just hoping to get one mailed to me or something and could not find the in game GM ticket.

**Steps to reproduce the problem**:

1. Open one of those bags from the Metzen event in Org bank maybe?
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

No proof, as I did not think to screenshot it at the time.

**Comment by TeridanDW** [comment]
Hello and thank you for the report @Lafadin,
I've tested both https://wowgaming.altervista.org/aowow/?item=17685 and https://wowgaming.altervista.org/aowow/?item=21216 and got item https://wowgaming.altervista.org/aowow/?item=21215 from both of them.

I would like to ask if someone else got this issue?

![wowscrnshot_122317_203730](https://user-images.githubusercontent.com/6797611/34321875-347dc8f6-e821-11e7-9fe7-45ca84797bce.jpg)
![wowscrnshot_122317_203701](https://user-images.githubusercontent.com/6797611/34321876-3498c6a6-e821-11e7-94ba-149af410eb47.jpg)

Closing until there is more information or a proof for the issue.
